poly-R.el

```diff
@@ -95,6 +95,10 @@ templates."
 ;;;###autoload (autoload 'poly-markdown+r-mode "poly-R")
 (define-polymode poly-markdown+r-mode poly-markdown-mode :lighter " PM-Rmd"
   :innermodes '(:inherit poly-r-markdown-inline-code-innermode))
+;;;###autoload (autoload 'poly-gfm+r-mode "poly-R")
+(define-polymode poly-gfm+r-mode poly-markdown+r-mode 
+  :lighter " PGFM-Rmd"
```

Comment:
PM stands for polymode, and I find PGFM a bit cryptic. I wonder if it would make sense to use something like PM-Rgfmd instead? Or maybe PM-GFM-Rmd, though it become too long then. Or PM-Rmd(gf)? Or just GFM-Rmd?

Reply:
Agreed. I think `PM-Rmd(gfm)` better conveys the idea that this is a variant of the original `PM-Rmd` mode using the `gfm` version of markdown.

Much as I have enjoyed this dive into polymode, it seems like the few additional features provided by `gfm` could just be included as options in markdown-mode, which would have saved a lot of effort!

Reply:
Hmm, not sure what the exact difference is, but I hope the author of markdown mode made this split for a good reason. We just follow with that.